app/src/test/java/at/gammastrahlung/monopoly_app/game/GameDataTest.java

```diff
@@ -118,6 +120,24 @@ void webSocketConnected() {
         assertTrue(gameData.isWebSocketConnected());
     }
 
+    @Test
+    public void testLastMessageTypeGetterAndSetter() {
```

Comment:
This has nothing to do with your feature

Reply:
Yes, but SonarCloud told me that I have to do a Test on this, which I did.

Reply:
If you for **example**delete the code accidentally and do Ctrl+z it will be counted as your code.